README.md

```diff
@@ -220,8 +220,8 @@ These plugins are hosted at <https://github.com/open-telemetry/opentelemetry-js-
 - [@opentelemetry/plugin-ioredis][otel-contrib-plugin-ioredis]
 - [@opentelemetry/plugin-express][otel-contrib-plugin-express]
 - [@opentelemetry/plugin-dns][otel-contrib-plugin-dns]
```

Comment:
shouldn't we link to instrumentation instead of plugin now ?

Reply:
I'm removing the unused parts only, once we have mysql instrumentation we can start removing plugins and updating links to point to instrumentations until then we should not change it. Notice `-` doesn't mean I changed anything in this line it is just points list sign.

Reply:
Yeah i know that you didn't change it, i was just asking